src/main/java/com/auth0/json/mgmt/ResourceServer.java

```diff
@@ -0,0 +1,183 @@
+        this.id = id;
+        this.name = name;
+        this.identifier = identifier;
+        this.scopes = scopes;
+        this.signingAlg = signingAlg;
+        this.signingSecret = signingSecret;
+        this.allowOfflineAccess = allowOfflineAccess;
+        this.skipConsentForVerifiableFirstPartyClients = skipConsentForVerifiableFirstPartyClients;
+        this.tokenLifetime = tokenLifetime;
+        this.verificationKey = verificationKey;
+        this.verificationLocation = verificationLocation;
+        this.isSystem = isSystem;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Boolean getIsSystem() {
+        return isSystem;
+    }
+
+    public void setIsSystem(Boolean system) {
+        isSystem = system;
+    }
+
+    public String getIdentifier() {
+        return identifier;
+    }
+
+    public void setIdentifier(String identifier) {
+        this.identifier = identifier;
+    }
+
+    public List<Scope> getScopes() {
+        return scopes;
+    }
+
+    public void setScopes(List<Scope> scopes) {
+        this.scopes = scopes;
+    }
+
+    public String getSigningAlg() {
+        return signingAlg;
+    }
+
+    public void setSigningAlg(String signingAlg) {
+        this.signingAlg = signingAlg;
+    }
+
+    public String getSigningSecret() {
+        return signingSecret;
+    }
+
+    public void setSigningSecret(String signingSecret) {
+        this.signingSecret = signingSecret;
+    }
+
+    public Boolean getAllowOfflineAccess() {
+        return allowOfflineAccess;
+    }
+
+    public void setAllowOfflineAccess(Boolean allowOfflineAccess) {
+        this.allowOfflineAccess = allowOfflineAccess;
+    }
+
+    public Boolean getSkipConsentForVerifiableFirstPartyClients() {
+        return skipConsentForVerifiableFirstPartyClients;
+    }
+
+    public void setSkipConsentForVerifiableFirstPartyClients(Boolean skipConsentForVerifiableFirstPartyClients) {
+        this.skipConsentForVerifiableFirstPartyClients = skipConsentForVerifiableFirstPartyClients;
+    }
+
+    public Integer getTokenLifetime() {
+        return tokenLifetime;
+    }
+
+    public void setTokenLifetime(Integer tokenLifetime) {
+        this.tokenLifetime = tokenLifetime;
+    }
+
+    public String getVerificationKey() {
+        return verificationKey;
+    }
+
+    public void setVerificationKey(String verificationKey) {
+        this.verificationKey = verificationKey;
+    }
+
+    public String getVerificationLocation() {
+        return verificationLocation;
+    }
+
+    public void setVerificationLocation(String verificationLocation) {
+        this.verificationLocation = verificationLocation;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Remove this method as we chose not to provide it on the other entities.

Reply:
done